tools/config/config.cpp

```diff
@@ -2,23 +2,120 @@
 #include "config.h"
 #undef __APICALL_EXTERN
 
+#include "logging.h"
+#include <fstream>
+
+LOG_DEFINE_MODULE(Config);
```

Comment:
It throws an undefined symbol error on every logging call there.

Reply:
// No logging here (logging depends on config )
// Can be put together with logging if needed

replace the traces with printf and remove logging. At above as reminder.

Reply:
Oooh, right. I forgot about it, lol...